app/templates/ui/app/search/search.controller.js

```diff
@@ -3,21 +3,54 @@
   'use strict';
 
   angular.module('app.search')
+    .factory('SearchModel', SearchModel)
     .controller('SearchCtrl', SearchCtrl);
 
-  SearchCtrl.$inject = ['$scope', '$location', 'userService', 'MLSearchFactory'];
+  SearchModel.$inject = ['MLSearchFactory'];
+  SearchCtrl.$inject = ['$scope', '$location', 'userService', 'SearchModel'];
 
   // inherit from MLSearchController
   var superCtrl = MLSearchController.prototype;
   SearchCtrl.prototype = Object.create(superCtrl);
 
-  function SearchCtrl($scope, $location, userService, searchFactory) {
-    var ctrl = this;
+  function SearchModel(searchFactory) {
```

Comment:
More of a useful thing SE's could use to preserve state in other controllers as well. Not really necessary if difficult..